pom.xml

```diff
@@ -22,7 +22,25 @@
 	<properties>
 		<jenkins.version>2.7.3</jenkins.version>
 		<java.level>8</java.level>
+		<skip.unit.tests>false</skip.unit.tests>
+		<skip.integration.tests>true</skip.integration.tests>
 	</properties>
+
+	<profiles>
+		<profile>
+			<id>dev</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+			</activation>
+		</profile>
+		<profile>
+			<id>integration-test</id>
+			<properties>
+				<skip.integration.tests>false</skip.integration.tests>
+				<skip.unit.tests>true</skip.unit.tests>
+			</properties>
+		</profile>

```

Comment:
Doesn't look like this profile is set in the `azure-pipelines.yml`, so I don't think these tests are running in CI yet.

Reply:
Changed pom to remove profiles. Integ tests are now run in the build, because it is mapped to 'Verify goal'.